docs/62_dandi_cli.md

```diff
@@ -0,0 +1,48 @@
+For data management (predominately `upload`, `download` and `validation` of data 
```

Comment:
@jwodder @yarikoptic would be great if you could review

For reference to easily read in staging setting: https://aquamarine-profiterole-e20e84.netlify.app/62_dandi_cli/

Reply:
Thanks, Aaron.  Just noting that given your recent developments to push `lincbrain-cli` changes upstream to the `dandi-cli` in https://github.com/dandi/dandi-cli/pull/1519, we will need to update these instructions.